### PR TITLE
"build dev-install" overwrites existing symbol links

### PR DESCRIPTION
**Pull request by ChengTian**
This makes it convenient to switch between different local dnx repos

@davidfowl @victorhurdugaci

### CODE REVIEW

File: makefile.shade [hunk 1]
```diff
@@ -1138,11 +1138,18 @@ functions @{
...
         {
-            // Nothing to do, just return
-            return;
+            if (IsMono)
+            {
+                Exec("rm", "-rf " + destinationDir);
+            }
+            else
+            {
+                Exec("cmd", "/C rmdir " + destinationDir);
```
Comment (victorhurdugaci) [inline makefile.shade]: 1. Why don't you call `rmdir` directly?
2. Why not use `Directory.Delete` ?
3. Will `rm -rf` remove the contents of the symlink or just the symlink itself?

Comment (davidfowl) [inline makefile.shade]: Use 2.

**Comment by ChengTian** [comment]
Thanks for the feedback. Please check the updated patch.

**Comment by victorhurdugaci** [comment]
Looks good. :shipit:

**Pull request merged**